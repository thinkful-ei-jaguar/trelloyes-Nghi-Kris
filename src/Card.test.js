import React from 'react';
import reactDOM from 'react-dom';
import renderer from 'react-test-renderer';

import Card from './Card';

// Snapshot Test
it('Renders Card UI', () => {
    const jsonFile = renderer.create(<Card title='cardA' content='stuff'/>).toJSON();
    expect(jsonFile).toMatchSnapshot();
})

// Smoke Test
it('Card is rendering',() => {
    const div = document.createElement('div');
    reactDOM.render(<Card />, div);
    // Clean up
    reactDOM.unmountComponentAtNode(div);
})