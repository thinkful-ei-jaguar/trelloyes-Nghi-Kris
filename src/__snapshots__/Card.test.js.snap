// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders Card UI 1`] = `
<div
  className="Card"
>
  <h3>
    cardA
  </h3>
  <p>
    stuff
  </p>
</div>
`;
